lib/Elevate/Blockers/Repositories.pm

```diff
@@ -345,15 +365,37 @@ sub _system_update_check ($self) {
         WARN("Your system is not up to date please run: /usr/bin/yum update");
 
         my $is_blocker;
-        my $output = $out->{stdout} // [];
+        my $output       = $out->{stdout} // [];
+        my $ood_packages = {
+            'outdated_packages_count' => 0,
+            'outdated_packages'       => [],
+            'repositories'            => {},
+        };
         foreach my $line (@$output) {
             next if $line =~ qr{^\s+$};
             next if $line =~ qr{^kernel};    # do not block if we need to update kernel packages
             $is_blocker = 1;
-            last;
+
+            if ( my ( $pkg_name, $pkg_version, $pkg_repo ) = $line =~ qr{^([\w.-]+)\s+([\w.:-]+)\s+([\w-]+)$}x ) {
+                $ood_packages->{'outdated_packages_count'}++;
+                push( @{ $ood_packages->{'outdated_packages'} }, { 'name' => $pkg_name, 'version' => $pkg_version, 'repository' => $pkg_repo } );
+                push( @{ $ood_packages->{'repositories'}->{$pkg_repo}->{'packages'} }, { 'name' => $pkg_name, 'version' => $pkg_version } );
+            }
         }
 
         # not a blocker when only kernels packages need to be updated
+        my $blocker_id = ref($self) . '::YumOutOfDate';
+        $self->has_blocker(
+            Cpanel::JSON::canonical_dump(
+                {
+                    'name'  => $blocker_id,
+                    'error' => 'YUM reports there are out of date packages.',
+                    'data'  => $ood_packages,
```

Comment:
https://tardis.fandom.com/wiki/Ood?file=Ood_Sigma_%26_Co.jpg

Reply:
did you mean odd?

Reply:
Nope, ood is correct as I defined it earlier as `ood_packages` for "out of date packages"